code/GraphQL/Resolvers/AssetAdminResolver.php

```diff
@@ -126,7 +127,7 @@ public static function resolveDeleteFiles($object, array $args, $context, Resolv
         $deletedIDs = [];
         $member = UserContextProvider::get($context);
         foreach ($files as $file) {
-            if ($file->canArchive($member)) {
+            if (Deprecation::withNoReplacement(fn() => $file->canArchive($member))) {
```

Comment:
```suggestion
            $canArchive = Deprecation::withNoReplacement(fn() => $item->canArchive());
            if ($canArchive) {
```
Makes this a lot easier to read

Reply:
Done